hls-plugin-api/src/Ide/Types.hs

```diff
@@ -873,7 +873,7 @@ data PluginCommand ideState = forall a. (FromJSON a) =>
 type CommandFunction ideState a
   = ideState
   -> a
-  -> LspM Config (Either ResponseError Value)
+  -> LspM Config (Either ResponseError (Value |? Null))
```

Comment:
tbh, since they're represented the same, I'm not sure this really adds much except making things more complex for the users

Reply:
I would say less complex. LSP Types are used everywhere, and now command users don't need to use Aeson if they don't use it anywhere else (Pretty much every command returns null).

Reply:
Good point. Is there any reason why we can't use Aeson.Null for Null in lsp-types, perhaps by reexporting it instead of declaring our own? that would remove a source of name conflicts.

Reply:
Okay right, because it's only a constructor, not a type in its own right.